js/stalk.js

```diff
@@ -16,6 +16,12 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 var c;
+var dreamjub_url = 'http://localhost:8000/';
+var dreamjub_client_id = 'uXDaB3j1n8DiOU8WXjwCa2gbKqLveieoIDpjFa1d';
+
+// var dreamjub_url = 'https://leadership.jacobs.university/';
```

Comment:
Should this be pointing to the staging version? Not sure.

Reply:
Well, it was a comment for testing.

Reply:
In production, it actually points to the jacobs.university instance, but I didn't want to put the key in this repository.

Reply:
Fair enough.